service/server/server.go

```diff
@@ -74,7 +119,7 @@ func (os *OracleServer) StartServer(ctx context.Context, host, port string) erro
 		<-ctx.Done()
 		os.logger.Info("context cancelled, closing oracle")
 
-		os.Close()
+		_ = os.Close()
 		return nil
```

Comment:
Do we not care about this err?

Reply:
It's only called when the context is closed and the server is shutting down, so I think that's why it's being thrown out. I've just explicitly ignored it because my editor complains about silently missing it.

We could choose to add a log here I guess. Not sure how noisy that would be--some things like the http.Server always return errors when they shutdown because of context being cancelled I believe.